.github/workflows/update.yaml

```diff
@@ -0,0 +1,47 @@
+name: Update
+
+# FIXME: Delete when the PR is ready
+on:
+  push:
+    branches: 
+      - "gh-actions-update"
+
+# Enable for daily auto-updates
+# on:
+#     schedule:
+#     - cron: 07 8 * * * # 08:07 UTC is 00:07 PST
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  auto-update:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+      
+      - name: Install dependencies
+        run: go get -t -d -v ./...
```

Comment:
Add a `-mod=readonly` bit like we use in `p1`.

Reply:
Using `go get` in module mode will try to update the `go.mod` (and I believe fail with `-mod=readonly`). We should change this to be explicit about using `go install` for any dependencies that need to be installed (do we have any?). Otherwise any non-`get` command will download the modules automatically.

Reply:
What should go here instead?

Reply:
Remove it. If there’s something in `make update` that needs it, we’ll find out!

Reply:
👍